apps/dav/lib/Connector/Sabre/AnonymousOptionsPlugin.php

```diff
@@ -67,7 +67,7 @@ public function handleAnonymousOptions(RequestInterface $request, ResponseInterf
 		$emptyAuth = $request->getHeader('Authorization') === null
 			|| $request->getHeader('Authorization') === ''
 			|| trim($request->getHeader('Authorization')) === 'Bearer';
-		$isAnonymousOption = $request->getMethod() === 'OPTIONS' && $emptyAuth;
+		$isAnonymousOption = $request->getMethod() === 'OPTIONS' && $isOffice && $emptyAuth;
```

Comment:
```suggestion
		$isAnonymousOfficeOption = $request->getMethod() === 'OPTIONS' && $isOffice && $emptyAuth;
```

Maybe name it like that? To avoid confusion?